src/azure-cli/azure/cli/command_modules/sql/__init__.py

```diff
@@ -20,6 +20,17 @@ def __init__(self, cli_ctx=None):
 
     def load_command_table(self, args):
         from azure.cli.command_modules.sql.commands import load_command_table
+        from azure.cli.core.aaz import load_aaz_command_table
+        try:
+            from . import aaz
```

Comment:
I'm not sure if we have guideline to not use relative path for `import`. This might cause problems when packaging.

Reply:
Short answer is no. This is more of a personal preference. I am already using relative import in MSAL migration: https://github.com/Azure/azure-cli/blob/1d973cceb38980181eeaa45934497d2148a3e7b2/src/azure-cli-core/azure/cli/core/auth/identity.py#L16-L20